CPP-Examples.cpp

```diff
@@ -11,16 +11,17 @@ int main()
 
```

Comment:
[Banned C function detected (strncat)](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS108330.md#DS108330) 7 times
strncat adds the null terminator at character 'n + 1', rather than at the nth character. this frequently leads to the null terminator being added in the memory adjacent to the destination buffer, rather than in the destination buffer.

*Fix Guidance:* Use strcat_s or strlcat if possible. 


Suggestions:
[1] 
[2]